shared/downloads/jeos-27-64.ini

```diff
@@ -0,0 +1,6 @@
+[jeos-27-64]
+title = JeOS 27 x86_64
+url = http://assets-avocadoproject.rhcloud.com/static/jeos-27-64.qcow2.xz
```

Comment:
To host these new images, we'll have to go ahead and switch to our new server (which has a lot more storage).  So this means changing the URL.  Just a heads up because this will require changes and some wait time, as I prepare the new server.

Reply:
OK, no problem. Let me know when you have the urls.